Client/SafeBrowsing/GSafeBrowsing.swift

```diff
@@ -0,0 +1,466 @@
+// Copyright (c) 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import CommonCrypto
+import Shared
+
+private let log = Logger.browserLogger
+
+enum SafeBrowsingResult {
+    case safe
+    case dangerous(ThreatType)
+    case unknown
+}
+
+class SafeBrowsingClient {
+    private static let apiKey = "DUMMY_KEY"
```

Comment:
Is there a reason we need this at all since we go through Brave proxy?

Reply:
Yes, without it, it doesn't work. Apparently I must send DUMMY_KEY to the brave server and it replaces it with the correct key.